### PR TITLE
Only update the Truffle import if not shared with TruffleRuby as a source suite

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

File: tool/jt.rb [hunk 1]
```diff
@@ -519,7 +546,11 @@ def build(*options)
       sulong = Utilities.find_or_clone_repo('https://github.com/graalvm/sulong.git')
       chdir(sulong) do
         raw_sh 'git', 'pull'
-        mx 'sforceimports'
+        # Only update the Truffle import if not shared with TruffleRuby as a source suite
```
[the remark this answers is not in the document]
Comment (eregon) [reply tool/jt.rb]: In the case both TruffleRuby or Sulong use the same Truffle (source) suite, we don't want to update Truffle to the revision in Sulong's suite.py, as we prefer TruffleRuby's truffle revision in that case (same in CI, same in the downstream Ruby test in Sulong).

But, if TruffleRuby or Sulong uses a binary import of Truffle, then they use separate Truffle suites and then we want the Truffle suite used by Sulong to have the matching revision with Sulong's suite.py.

When building TruffleRuby with `jt build` we always `mx sforceimports` so the Truffle version we build against is the one in our `mx.truffleruby/suite.py`.

**Comment by eregon** [comment]
@nirvdrum It doesn't look like mx exposes this information.
We could parse the output of `mx sversions` but parsing the `env` file seems easier/cleaner.

File: tool/jt.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisseaton) [reply tool/jt.rb]: But we don't depend on Sulong?

[the remark this answers is not in the document]
Comment (eregon) [reply tool/jt.rb]: No, yet if you use no binary suites (MX_BINARY_SUITES) both will use the same Truffle checkout.
Which means we end up rebuilding Truffle on each `jt build`/`jt build sulong` alternation and we build Sulong locally in a configuration that is not tested in our CI.

**Pull request merged**